README.md

```diff
@@ -60,13 +60,19 @@ with config files from '2020.08.07.ini', '2021.01.26.ini', '2021.02.21.ini' or 2
 
 ### Training a tech tagging model
 
-You can train and save a model to classify grants as being to do with tech or not (see definitions for this in [Finding_Tech_Grants.md](docs/Finding_Tech_Grants.md)) by running:
+You can train and save a model, or several models, to classify grants as being to do with tech or not (see definitions for this in [Finding_Tech_Grants.md](docs/Finding_Tech_Grants.md)) by creating a config file which contains various training arguments for the experiments. The main arguments in this config file are:
 
+- `training_data_file`, e.g. `data/processed/training_data/210308/training_data.csv`, the training data file.
+- `vectorizer_types`, e.g. `['count', 'tfidf', 'bert', 'scibert']`, the vectorizers you want to use.
+- `classifier_types`, e.g. `['naive_bayes', 'SVM', 'log_reg']`, the classifier types you want to use.
+
+Then by running 
 ```
-python nutrition_labels/grant_tagger.py --training_data_file data/processed/training_data.csv --vectorizer_type count --relevant_sample_ratio 1 --model_type naive_bayes --bert_type bert
+python nutrition_labels/grant_tagger.py --config_path configs/train_model/2021.03.16.ini
```

Comment:
nice way to run models and have them reproducible. in case of interest you can define an entry point like I have done here https://github.com/wellcometrust/grants_tagger/blob/f269800b12ea6081d545a97a573ed44f01c6cb80/setup.py#L26
then you can all `grants_tagger --config_path configs/train_model/2021.03.16.ini`

as far as I know you have multiple entry points or you can also add a CLI like I have done here using Typer https://github.com/wellcometrust/grants_tagger/blob/master/grants_tagger/__main__.py (you can also use argparse) and then introduce commands for train, predict etc so you can do
`grants_tagger train --config_path ...` or `grants_tagger evaluate ...`